data_definition_postgres.sql

```diff
@@ -4,12 +4,15 @@ DROP TABLE IF EXISTS recipe_category;
 DROP TABLE IF EXISTS ingredient_ethical_problem;
 DROP TABLE IF EXISTS ingredient_alternative;
 
+
 -- Then drop tables that are depended on
 DROP TABLE IF EXISTS ingredient;
 DROP TABLE IF EXISTS recipe;
 DROP TABLE IF EXISTS category;
 DROP TABLE IF EXISTS account;
 DROP TABLE IF EXISTS ethical_problem;
+DROP TABLE IF EXISTS ethical_explaination;
```

Comment:
Why do we need a separate table for the explanation? Do we expect a single ethical issue to have multiple explanations, or a single explanation to be applicable to multiple issues? To me, the explanation can just be a field in the `ethical_problem` table.

Also, "explaination" is a typo. Please change it to "explanation" throughout.

Personally, I prefer the name "description", but I am fine either way.

Reply:
@anthonyliuhao i fixed the typos.  For the separate table, I chose this way because one ethical issue could have more than one problem (i.e deforestation could have water issues and land issues). I think for this assignment will be sticking with one, but i wanted to make it flexible.